cmd/expiration-mailer/main.go

```diff
@@ -292,96 +293,124 @@ func (ds durationSlice) Swap(a, b int) {
 
 const clientName = "ExpirationMailer"
 
+type config struct {
+	Mailer struct {
+		cmd.ServiceConfig
+		cmd.DBConfig
+		cmd.SMTPConfig
+
+		From    string
+		Subject string
+
+		CertLimit int
+		NagTimes  []string
+		// How much earlier (than configured nag intervals) to
+		// send reminders, to account for the expected delay
+		// before the next expiration-mailer invocation.
+		NagCheckInterval string
+		// Path to a text/template email template
+		EmailTemplate string
+	}
+
+	Statsd cmd.StatsdConfig
+
+	Syslog cmd.SyslogConfig
+}
+
 func main() {
-	app := cmd.NewAppShell("expiration-mailer", "Sends certificate expiration emails")
-
-	app.App.Flags = append(app.App.Flags, cli.IntFlag{
-		Name:   "cert_limit",
-		Value:  100,
-		EnvVar: "CERT_LIMIT",
```

Comment:
@ccppuu I forgot to mention that in PR comment. We did remove the `WFE_LISTEN_ADDR` in the WFE. 

The code for this flag did not make much sense (or I have misunderstood). It set a default value to 100, then checked to see if the value was greater than 0, if it was then it set `c.Mailer.CertLimit`.